### ISSUE TITLE
[compdat] STOP is not a valid state for a completion

### ISSUE THREAD

**Issue by asnyv** [equinor/ecl2df#222]
When using something like 
```
WELOPEN 
SOMEWELL STOP /
/
```
The connections/completions get the state `STOP` from `ecl2df`, whilst they in reality are `OPEN`. Crossflow can still occur, well is closed above perforations.

Similarily, if applied directly to connections/completions
```
WELOPEN 
SOMEWELL STOP 1 1 4 6 /
/
```
They also get the state `STOP`, whilst in Eclipse they would in reality be `SHUT`

`STOP` is only a state in Eclipse for wells, not completions/connections

**Comment by berland** [comment]
Is this a problem with the dataframe produced, or a problem with how to interpret the dataframe?

**Comment by asnyv** [comment]
My personal meaning is that it is an issue with how the dataframe is produced, as it is interpreted directly as stated in the WELOPEN keyword, and not in the same way the Eclipse documentation states that Eclipse handles these operations.

Also, have just tested quickly, but seems like `WELOPEN` in `ecl2df` operates on all connections for a well, even when specifying connections? Need to test it a bit more before I declare it as a bug, but seems like it doesn't work as intended (unless recently fixed)

**Comment by berland** [comment]
This is a limitation of the dataframe concept of connection in ecl2df, see #298 which adds a warning for this scenario. It is maybe still solvable in the compdat module without adding an extra layer explicitly giving the "well state", by a more advanced `applywelopen`.

**Comment by berland** [comment]
I fully agree this is a bug. `ecl2df.compdat` should fully ignore things like
```
WELOPEN 
SOMEWELL STOP /
/
```

and when STOP is used on specified connections, then it should be translated to SHUT. See #298 for documentation excerpt.
